internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -6131,7 +6238,14 @@ func TestHandleTSpendVotes(t *testing.T) {
 	// Chain that returns an older header in HeaderByHeight.
 	chainOldBlock := new(testRPCChain)
 	*chainOldBlock = *chainVotes
-	chainOldBlock.headerByHash.Height = 1000
+	header, err := chainOldBlock.HeaderByHash(nil)
+	if err != nil {
+		t.Fatalf("[HeaderByHash] unexpected error: %v", err)
```

Comment:
The header in the message really isn't needed.  Test failures always log the line they came from and the test function that was running, so it doesn't provide any extra information.  That said, it's not a big deal, so I'll approve anyway.  Feel free to change it if you want.